tools/generate-pdfs.sh

```diff
@@ -18,26 +18,32 @@ set -ex
 
 function generate_pdfs_from_md() {
 	inputMdFile=$1
+	
 	if ! [ -f "$inputMdFile" ]; then
 		echo "**** WARNING! $inputMdFile does not exist. Please enter valid file path."
 		echo "**** WARNING! Please make sure to run tools/generate-intrinsics-specs.sh before building the PDFs."
 		exit 1
 	fi
 
 	outputPdfFile=$2
-	configYamlFile=$3
-	headingLineNum=$(awk '/<!---END_OF_HTML_HEADER--->/ { print NR; exit }' $inputMdFile)
+	geometryForIntrinsics='\\newgeometry{landscape,top=3.7cm,bottom=2.7cm,left=1cm,right=1cm,headsep=1.5cm,footskip=.5cm}'
 
-	tail -n +$headingLineNum $inputMdFile | \
-	pandoc --template=tools/acle_template.tex --metadata-file=$configYamlFile -o $outputPdfFile
+	# This line replaces the ToC declaration in the md files with a blank space.
+	# ":a;N;$!ba;" is at the start so sed could recognise newline.
+	# The rest is a regular expression.
+	# The second replacement string is being used to format the "List of Intrinsics"
+	# sections, which contain large longtables.
+	sed -u ':a;N;$!ba;s/\*\sTOC\n{*{:toc}}*//' $inputMdFile | \
+	sed -u "s/#\sList\sof\sIntrinsics/${geometryForIntrinsics}\n# List of Intrinsics/" | \
```

Comment:
I reall like this idea. However, now that I see the sequence of `sed` in bash, I think it gets a bit... hacky.

How about change the MD templates as follows:

```
titling stuff, followed by the jekyll command...

* TOC
{:toc}

... then more content (license and so on), and finally....

{latex_geometry_for_intrinsic}
# List of intrinsics

{intrinsic_list}
```

Then, modify the python script so that:

1. The `* TOC+ {:toc}` is removed
2. The variable `{latex_geometry_for_intrinsics}` is mapped to an empty string when targeting the markdown files of the specs, and it is mapped to the `\newgeometry{whatever}` when targeting the pdf.

I know that what I am proposing is totally equivalent to what you have already implemented, but I think it will be easier to handle in python than in bash, especially if it turns out we have to add more substitutions...

Reply:
I just realized that item 1 (remove TOC in the python script) is not doable because we don't run the python script on all the specs... so please leave the `sed` commands here. But please use a comment that gives some hint of what is going on for the replacement, not the title:

```
titling stuff, followed by the jekyll command...

* TOC
{:toc}

... then more content (license and so on), and finally....

<!--- latex_geometry_conf -->
# List of intrinsics

{intrinsic_list}

```

In the `sed` invocation replace the `<!--- latex_geometry_conf -->`